include/global.fieldmap.h

```diff
@@ -1,6 +1,13 @@
 #ifndef GUARD_GLOBAL_FIELDMAP_H
 #define GUARD_GLOBAL_FIELDMAP_H
 
+#define MAP_IMPASSABLE_MASK 0x0C00
```

Comment:
I'd change these to use a `METATILE_` prefix, instead of `MAP_`.  
`METATILE_COLLISION_MASK`
`METATILE_ID_MASK`
`METATILE_ID_UNDEFINED`
`METATILE_ELEVATION_SHIFT`
`METATILE_COLLISION_SHIFT`
`METATILE_ELEVATION_MASK`